spec/mainadditions.html

```diff
@@ -11,6 +11,26 @@ <h1>Amendments to the ECMAScript® 2021 Language Specification</h1>
     </p>
   </emu-note>
 
+  <emu-clause id="sec-literals-numeric-literals">
+    <h1><a href="https://tc39.es/ecma262/#sec-literals-numeric-literals">Numeric Literals</a></h1>
+    <emu-note type="editor">
+      <p>
+        No changes, but these productions must be present for symbol references because biblio.json contents are not propagated to grammarkdown.
+      </p>
+    </emu-note>
+    <emu-grammar type="definition">
+      DecimalDigits[Sep] ::
+        DecimalDigit
+        DecimalDigits[?Sep] DecimalDigit
```

Comment:
I realize this isn't used in this spec, but I think we should probably keep
```
  [+Sep] DecimalDigits[+Sep] NumericLiteralSeparator DecimalDigit 
```
to avoid confusion

Reply:
Done.